docs/about/announcement.md

```diff
@@ -0,0 +1,34 @@
+# Project Announcement
```

Comment:
I don't think this page is relevant in documentation for developers, it has so many details that are irrelevant now.

Reply:
should i delete it at all or put it somewhere else?

Reply:
delete it, we'll keep it in the wiki for historical reasons

Reply:
OK, although i would rather suggest keep it somewhere else and disable wiki completely.